checks/permissions.go

```diff
@@ -613,3 +613,128 @@ func isReleasingWorkflow(workflow *actionlint.Workflow, fp string, dl checker.De
 
 	return fileparser.AnyJobsMatch(workflow, jobMatchers, fp, dl, "not a releasing workflow")
 }
+
+// TODO: remove when migrated to raw results.
+// Should be using the definition in raw/packaging.go
+func isPackagingWorkflow(workflow *actionlint.Workflow, fp string, dl checker.DetailLogger) bool {
```

Comment:
Nit: `isPackagingWorkflow` can be made part of `fileparser` package and be shared across both the `permissions` and `packaging` checks?

Reply:
in the future yes. For now it's not needed because it's going to be removed in https://github.com/ossf/scorecard/pull/1912. The code here is a temporary solution until the permission check is migrated.